projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cds/profile-tag.ts

```diff
@@ -14,3 +16,11 @@ export const profileTagHelper = {
   },
   profileTagScriptResponse: {},
 };
+
+export function grantConsent() {
```

Comment:
FYI, you can add this function to the above const, this way someone does `profileTagHelper. grantConsent`

Reply:
👍 will do